garage/tf/policies/gaussian_mlp_policy.py

```diff
@@ -4,14 +4,14 @@
 from garage.core import Serializable
 from garage.misc import logger
 from garage.misc.overrides import overrides
-from garage.spaces.tf import Box as TfBox
 from garage.tf.core import LayersPowered
 from garage.tf.core import MLP
 import garage.tf.core.layers as L
 from garage.tf.distributions import DiagonalGaussian
 from garage.tf.misc import tensor_utils
 from garage.tf.misc.tensor_utils import enclosing_scope
 from garage.tf.policies import StochasticPolicy
+from garage.tf.spaces import Box as TfBox
```

Comment:
Not necessary to call this TfBox -- we're in the TF subtree and we're not using any other kind of Box.